reference/configuration.rst

```diff
@@ -7,14 +7,28 @@ There are various configuration parameters that affect the behaviour of Citus. T
 
 The rest of this reference aims at discussing Citus specific configuration parameters. These parameters can be set similar to PostgreSQL parameters by modifying postgresql.conf or `by using the SET command <http://www.postgresql.org/docs/9.6/static/config-setting.html>`_.
 
-Node configuration
+General configuration
 ---------------------------------------
 
 citus.max_worker_nodes_tracked (integer)
 $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
 
 Citus tracks worker nodes' locations and their membership in a shared hash table on the coordinator node. This configuration value limits the size of the hash table, and consequently the number of worker nodes that can be tracked. The default for this setting is 2048. This parameter can only be set at server start and is effective on the coordinator node.
 
+.. _enable_version_checks:
+
+citus.enable_version_checks (bool)
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+Upgrading Citus version requires a server restart (to pick up the new
+shared-library), as well as running an ALTER EXTENSION UPDATE command. The
+failure to execute both steps could potentially cause errors or crashes. Citus
+thus validates the version of the code and that of the extension match, and
+error's out if they don't.
+
+This value defaults to true, and is effective on the coordinator. In rare cases
+this parameter may set to false, thus disabling the check, and might be suitable
```

Comment:
I was initially confused by this sentence. What about something like:

In rare cases, complex upgrade processes may require setting this parameter to false.